gufe/tests/test_tokenization.py

```diff
@@ -232,6 +247,39 @@ def test_from_keyed_dict(self):
         assert recreated == self.cont
         assert recreated is self.cont
 
+    def test_to_keyed_chain(self):
+        assert self.cont.to_keyed_chain() == self.expected_keyed_chain
+
+    def test_from_keyed_chain(self):
+        recreated = self.cls.from_keyed_chain(self.expected_keyed_chain)
+        assert recreated == self.cont
+        assert recreated is self.cont
+
+    def test_to_json_string(self):
+        raw_json = self.cont.to_json()
+        # tuples are converted to lists in JSON so fix the expected result to use lists
+        expected_key_chain = [list(tok) for tok in self.expected_keyed_chain]
```

Comment:
Ah, I guess this is a slight ugly spot in our keyed chain representation, but philosophically I do prefer the tuple usage since each element of a keyed chain has exactly two elements of its own.

Reply:
I agree the tuple is nicer I'll just leave this here so others know whats going on.